### PR TITLE
Set up website

### PR DESCRIPTION
**Pull request by ShadyShawkat**
In this project, I did the following:

- [x] Add Home page (Mobile/Desktop).
- [x] Add About page (Mobile/Desktop).
- [x] Add Nav menu for the mobile.
- [x] Use Bootstrap 5 for styling.
- [x] Use SCSS for styling.
- [x] Implement dynamic elements on the Home page ( Featured Guests section ) using Javascript.
- [x] Show more featured guests on "More" button click.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+        <div class="d-flex justify-content-between align-item-center container-fluid px-0">
+          <a class="navbar-brand d-none d-md-flex pt-0 ps-3" href="./index.html">
+            <img src="./assets/images/logo.png" class="my-auto" alt="Logo" width="200" height="24">
+          </a>
+          <button
+            class="navbar-toggler px-3 mx-2"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbar"
+            aria-controls="navbar"
```
Comment (Davidosky007) [inline index.html]: * I suggest you make the navbar identical to that of the original design, Please note ✍️ that you are to follow the design guidelines which includes: Colors.
Typography: font face, size and weight.
Layout: composition and space between elements. Kindly compare and make the necessary changes 
## ORIGINAL 
* Your design is missing the top dark bar with links and social icons 
image
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
## YOURS 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @ShadyShawkat  👋, 

Good job so far! 💪 
* Professional looking readme file ✔️ 
* No linter errors ✔️ 
* Proper use of git-flow ✔️ 
* Descriptive commit messages ✔️ 
* Well written PR ✔️ 

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
* Kindly provide a link to a video presentation not more than 5mins  presenting the features of the project with a clear objective description of the project [Check project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Once%20you%20have,pull%20request%20description.)
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ShadyShawkat) [reply index.html]: Done.

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @ShadyShawkat 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**